Formula/openmama.rb

```diff
@@ -0,0 +1,42 @@
+class Openmama < Formula
+  desc "Open source high performance messaging API for various Market Data sources"
+  homepage "https://openmama.org"
+  url "https://github.com/finos/OpenMAMA/archive/OpenMAMA-6.3.1-release.tar.gz"
+  sha256 "43e55db00290bc6296358c72e97250561ed4a4bb3961c1474f0876b81ecb6cf9"
+  license "LGPL-2.1-only"
+
+  depends_on "cmake" => :build
+  depends_on "apr"
+  depends_on "apr-util"
+  depends_on "libevent"
+  depends_on "ossp-uuid"
+  depends_on "qpid-proton"
+  uses_from_macos "flex" => :build
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-DAPR_ROOT=/usr/local/opt/apr", "-DINSTALL_RUNTIME_DEPENDENCIES=OFF", *std_cmake_args
+      system "make", "-j", "install"
```

Comment:
Brew already passes `-j`

```suggestion
      system "make", "install"
```

Reply:
Cool will do

Reply:
All sorted now.